cmd/dbconv/dbconv/dbconv.go

```diff
@@ -0,0 +1,184 @@
+package dbconv
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+var unfinishedConversionCanaryKey = []byte("unfinished-conversion-canary-key")
+
+func UnfinishedConversionCheck(db ethdb.KeyValueStore) error {
+	unfinished, err := db.Has(unfinishedConversionCanaryKey)
+	if err != nil {
+		return fmt.Errorf("Failed to check UnfinishedConversionCanaryKey existence: %w", err)
+	}
+	if unfinished {
+		return errors.New("Unfinished conversion canary key detected")
+	}
+	return nil
+}
```

Comment:
UnfinishedConversionCheck and key should move away from /cmd.. probably to dbutil?